Travel Fund - Promotion to Stage-1 (#278)
openjs-foundation/cross-project-council: proposals/stage-1/TRAVEL_FUND/MEMBER_TRAVEL_FUND.md

# OpenJS Foundation Membership Travel Fund

## Purpose

To establish and administer a fund for members of the OpenJS Foundation to travel and spread knowledge about and support the Javascript ecosystem and the Foundation.

//TODO update link
### [Current Year Fund](https://github.com/nodejs/admin/blob/master/TravelFunds/2019.md)

//TODO update link
### [2018 funds](https://github.com/nodejs/admin/blob/master/TravelFunds/2018.md)

## Requirements

* Candidates must be an active participant of one of the projects of the OpenJS Foundation. //TODO clarify?
* Those requesting funds must indicate that they do not have funding available from another source, such as an employer or
the event itself that might cover costs for presenters.

## Process

Requests must be made prior to the start of travel. Requests submitted once travel is complete will not be approved.

It is strongly recommended that you submit your request far enough in advance to get approval before making relevant expenditures.
Review of requests will take at least 72 hours and may take considerably longer. Note, with limited funds available, approval is not

> guaranteed. If a request is rejected, reimbursement will not be issued.The Foundation cannot pre-pay for travel expenditures.

Reviewer: ```suggestion
guaranteed. If a request is rejected, reimbursement will not be issued. The Foundation cannot pre-pay for travel expenditures.
```

take longer for us to process.  You can email a zip file of your receipts and
the completed [Expense Report](./expense-report-template.xls?raw=true), renamed
as ExpenseReport-OpenJS-EVENTNAME-YOURNAME-YYYYMM.xls, to
[operations@openjsf.org](mailto:operations@openjsf.org).  Please send the file
as a spreadsheet, not as a PDF.  We will send it back to you for signature in
DocuSign.

### Important:

* Expense reports must be submitted within 30 days of event date. Any reports submitted after 30 days may not be reimbursed.
* Disbursements are generally processed within 30 days.
* We are unable to pre-pay for expenses.
* The Cross Project Council representative to the Board of Directors will report back amounts consumed from the travel allocation on a monthly basis to the Board.

## 2018 Board of Directors Allocation
The request from Technical Steering Committee and the Community Committee for a 2018 joint travel fund was
approved in the amount of $60,000. The $60,000 was fully utilized in 2018.

## 2019 Board of Directors Allocation

The request from Technical Steering Committee and the Community Committee for a 2019 joint travel fund

> was approved in the amount of $60,000.

Reviewer: The TSC and CommComm should be links, and we should name that they are entities in the Node.js project.

Author: Okay. Added links and clarified Node.js Foundation affiliation.

# OpenJS Foundation Membership Travel Fund

## Purpose

To establish and administer a fund for members of the OpenJS Foundation to travel and spread knowledge about and support the Javascript ecosystem and the Foundation.

//TODO update link
### [Current Year Fund](https://github.com/nodejs/admin/blob/master/TravelFunds/2019.md)

//TODO update link
### [2018 funds](https://github.com/nodejs/admin/blob/master/TravelFunds/2018.md)

## Requirements

> * Candidates must be an individual member of one of the projects of the OpenJS Foundation. //TODO clarify?

Reviewer 2: Individual member has specific meaning, maybe make this more general?

```suggestion
* Candidates must be an active participant of one of the projects of the OpenJS Foundation. //TODO clarify?
```

* Those requesting funds must indicate that they do not have funding available from another source, such as an employer or
the event itself that might cover costs for presenters.

## Process

Requests must be made prior to the start of travel. Requests submitted once travel is complete will not be approved.

It is strongly recommended that you submit your request far enough in advance to get approval before making relevant expenditures.
Review of requests will take at least 72 hours and may take considerably longer. Note, with limited funds available, approval is not
guaranteed. If a request is rejected, reimbursement will not be issued.The Foundation cannot pre-pay for travel expenditures.

> It is recommended to write a report after the trip.

Reviewer 3: Will it make sense to link a sample report here?

Author: That makes sense. I was actually wondering if we should remove the report entirely, or else enforce it as a requirement. If you look at the [last year of funded travel](https://github.com/nodejs/admin/blob/master/TravelFunds/2019.md), nobody filled out a report.

Reviewer 2: I'm +1 to removing the report

Author: Removed in latest commit.

## Visa Tips

> 

Reviewer 2: ```suggestion

// TODO make email alias for visa help
```

For all events, 3rd party or hosted Foundation events, it is recommended to apply for a visa at least 6 months prior to an event.

### For Industry Events

> If you need a visa for the country in which the OpenJSF event you are attending is being held,

Reviewer 4: ```suggestion
If you need a visa for the country in which the OpenJS Foundation event you are attending is being held,
```

[visainfo@openjsf.org](mailto:visainfo@openjsf.org).

## Requesting Travel Funds

* Open a pull request that adds an entry to the appropriate year's file
  * Include as many fields as possible:
    * The name of the event you plan to attend.
    * The location, dates of the event and where you will be traveling from.
    * The presentation you intend to give, if applicable.
    * The size of the stipend you wish to receive.
      * In the pull request description, explain how you calculated your
        stipend. For example, include estimates for airfare, hotel, other
        transportation, and so on. The total amount should be your stipend
        request.
      * In the PR description, mention @openjs-foundation/cpc.
    * Once the final amount spent is known, update the table again with that information.
    * Certain details of your request (such as the column for date sent to finance) may be unknown initially; these can be filled in and updated during the process of reconciliation.

## Approval

A request is approved when all of the following criteria are met:

* The pull request has been open for at least 72 hours.

> * The request has approval from at least four members of @openjs-foundation/cpc, including a voting representative for your project.

Author: We should discuss this. One of the current outstanding questions is who qualifies for fund usage. I added a change here, that a voting member for the applicant's project should approve it.

Reviewer 5: Not all projects have voting members.

Reviewer 4: I don't see why a project or its representative should need to positively approve a request, when they do have the opportunity to object to it.

Author: ok! I'll go back to earlier rule.

* No members of @openjs-foundation/cpc have objected to the request.
* The pull request has been merged. Any member may merge once other prerequisites have been met.

If a request is blocked by an objection from a member of @openjs-foundation/cpc, then the only way for the request
to be approved is for that committee to decide to approve the request using whatever decision-making process the committee uses when
consensus is not reached.

As examples, the purpose of prior sponsored travels includes attending TC39 meetings,

> attending [OpenJSF collaboration summits](https://github.com/nodejs/summit), and

Reviewer 4: ```suggestion
attending [OpenJS collaboration summits](https://github.com/nodejs/summit), and
```

mentoring [Code and Learn](https://github.com/nodejs/code-and-learn).

### Considerations for approval of a request

#### Impact

> * Preference is given to events hosted or sponsored by the OpenJSF Foundation. For example, OpenJSF collaboration summits and Diagnostics summits should be given precedence because they bring together members of the Foundation to help the Foundation projects advance.

Reviewer 4: ```suggestion
* Preference is given to events hosted or sponsored by the OpenJS Foundation. For example, OpenJS Collaboration summits and Diagnostics summits should be given precedence because they bring together members of the Foundation to help the Foundation projects advance.
```

  it with [operations@openjsf.org](mailto:operations@openjsf.org) on cc, and the
  Memo field set to "YOURNAME - TRIPNAME".

When we receive your report, we will send you a form via DocuSign to collect
your payment details.  At that point, you're done.

### Alternate process

If you decide not to use Expensify, the reimbursement process is manual and may
take longer for us to process.  You can email a zip file of your receipts and
the completed [Expense Report](./expense-report-template.xls?raw=true), renamed
as ExpenseReport-OpenJS-EVENTNAME-YOURNAME-YYYYMM.xls, to
[operations@openjsf.org](mailto:operations@openjsf.org).  Please send the file
as a spreadsheet, not as a PDF.  We will send it back to you for signature in
DocuSign.

### Important:

* Expense reports must be submitted within 30 days of event date. Any reports submitted after 30 days may not be reimbursed.
* Disbursements are generally processed within 30 days.
* We are unable to pre-pay for expenses.
* The Cross Project Council representative to the Board of Directors will report back amounts consumed from the travel allocation on a monthly basis to the Board.

> ## Trip report

Reviewer 2: If we remove the trip report request above we should remove this suggestions

Alternatively we can deep link

Author: Removed.
